packages/core/src/styles/Avatar.scss

```
@@ -0,0 +1,51 @@
+@import "./mixins";
+
+// -------------------------------------
+//   Component Block
+// -------------------------------------
+.#{$prefix}-avatar {
+    width: 2.2rem;
+    height: 2.2rem;
+    overflow: hidden;
+    margin: .2rem .45rem .2rem .25rem;
+    background-color: $c-gray;
+    flex-shrink: 0;
+
+    // ----------------------
+    //  <Avatar> variants
+    // ----------------------
+    &--square {
+        border-radius: 0;
+    }
+
+    &--rounded {
+        border-radius: 6px;
+    }
+
+    &--circle {
+        border-radius: 50%;
+    }
+
+    // ----------------------
+    //  Inner image
+    // ----------------------
+    > img {
```

You probably need a `&` here?
```suggestion
    & > img {
```